pkg/store/storepb/custom.go

```diff
@@ -177,3 +178,11 @@ func LabelsToString(lset []Label) string {
 	}
 	return "[" + strings.Join(s, ",") + "]"
 }
+
+func LabelSetsToString(lsets []LabelSet) string {
+	s := ""
+	for _, ls := range lsets {
```

Comment:
I think convention in Thanos is to use `strings.Join` but here we might use `bytes.Buffer.WriteString` I guess? I think we don't care about performance though as it's only for printing. Not a blocker, just `+` for strings feels wrong in Golang although it's heavily optimized nowadays.

Reply:
Replaced with `strings.Join` based approached.